share/doc/xml/combine_spectra.xml

```diff
@@ -361,6 +361,13 @@ SOURCE_BACKSCAL = (src_exp1*src_backscal1) + ... + (src_expN*src_backscalN) / (s
        </LIST>
 </ADESC>
 
+  <ADESC title="Changes in the script 4.15.1 (April 2023) release">
```

Comment:
Here I'm looking into the future. How do we usually mark that up for an unreleased version with no date yet?

Reply:
Well, sometimes we don't bother putting something in and hope we remember to add it in before a release. We sometimes use placeholder text (e.g. "XXX" rather than a month name). We plan to release something this month so this time you can use January ;-)